### PR TITLE
chore: rm deprecated ioutil package references

### PR DESCRIPTION
**Pull request by oscgu**
#### What type of PR is this?
/kind cleanup
/kind deprecation

#### What this PR does / why we need it:
This PR removes/resolves all references of the [ioutil](https://pkg.go.dev/io/ioutil#pkg-overview) pkg which has been deprecated since go1.16, except in the vendor and third_party directories, because I'm not sure if that should be done for those too.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/volume/fc/fc_util.go [hunk 1]
```diff
@@ -47,17 +47,37 @@ const (
...
+		info, err := e.Info()
+		if err != nil {
+			return nil, err
+		}
+		fi[i] = info
+	}
+
+	return fi, nil
 }
+
```
Comment (googs1025) [inline pkg/volume/fc/fc_util.go:67]: nit: If there are unnecessary spaces, we should remove them

**Comment by googs1025** [comment]
Can we split it into different PRs? This may make it difficult for reviewers to merge this PR.

**Comment by ffromani** [comment]
> Can we split it into different PRs? This may make it difficult for reviewers to merge this PR.

I second this. 1 PR per SIG (roughly, not dealbreaker if a PR span 2 SIGs) would be the fastest (better crosslink them somehow). One sweeping change is not wrong, it just makes the review longer because all the 6 involved SIGs (!) need to ack the change.

**Comment by oscgu** [comment]
@googs1025 @ffromani I'll close this PR then and open multiple smaller ones.

**Pull request closed without merging**